### PR TITLE
For fun!

### PR DESCRIPTION
**Pull request by DTloss**
This is second step mb?

### CODE REVIEW

**Comment by AmyLiyujie** [comment]
wow!

File: _pins/DTloss.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+===
+githubhandle:DTloss
```
Comment (Champ0505) [inline _pins/DTloss.json]: capital H on Handle

Comment (impappl) [inline _pins/DTloss.json]: We also need a space between the : and DTloss!

**Comment by hollenberry** [comment]
@DTloss @Champ0505 and @impappl are correct! You'll also need to update the === signs to look as dashes `---`

If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions!

**Comment by crichID** [comment]
@DTloss I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6692/commits/c85d847207904ae61cf4da739591e0732f781629. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**